lms/templates/main_django.html

```diff
@@ -21,6 +21,10 @@
   {% block headextra %}{% endblock %}
   {% render_block "css" %}
 
+  {% if THEME_ENABLED %}
+    {% include "theme-head-extra.html"|microsite_template_path %}
```

Comment:
This seems like an awkward usage pattern. Django's template lookup system is designed to handle the sort of directory lookup that the `microsite.get_template_path()` function does: just insert the directory at the front of the list of directories, and the template lookup system will find it first. For that matter, Mako's template lookup system is designed that way, too -- @nedbat and I are refactoring how this works in #8271. Could you take a look at what we're doing in that PR, and see if you can do the same sort of thing here? (Or we could just wait until it gets reviewed and merged, and then merge this in afterward.)

Also, you're doing a conditional `include` here. I think it would be better to do some kind of optional include: have the template system include the template file if it can find it, and if it can't find it, don't include anything. I implemented that in Mako in #8271, and I'm sure it would be possible to implement that in Django templates as well with a custom template tag.